src/components/ProjectDashboard/hooks/useProjectHeader.ts

```diff
@@ -51,3 +67,7 @@ export const useProjectHeader = (): ProjectHeaderData => {
     gnosisSafe,
   }
 }
+
+const stripHtmlTags = (html: string): string => {
+  return html.replace(/<[^>]*>/g, '')
```

Comment:
are we sure this is comprehensive?

Reply:
maybe not, but its good enough for now I think. We'll keep an eye on bugs. Realistically, people should be using tagline going forward anyway